game.go

```diff
@@ -303,28 +300,42 @@ type costumeConfig struct {
 	Y                float64 `json:"y"`
 	FaceRight        float64 `json:"faceRight"` // turn face to right
 	BitmapResolution int     `json:"bitmapResolution"`
+	Mode             string  `json:"mode"`
```

Comment:
costumeConfig is also used by sprite. so don't add `mode` in it.

Reply:
so，game support one map mode ? I thought  each scene would have its own mode。